python_modules/dagster/dagster/_core/definitions/asset_condition.py

```diff
@@ -64,15 +113,36 @@ class AssetConditionEvaluation(NamedTuple):
     def asset_key(self) -> AssetKey:
         return self.true_subset.asset_key
 
+    def get_candidate_subset(
```

Comment:
I don't see this used anywhere. Will it be?

Reply:
Ping on this in case you missed it. Asking because I'm wondering whether it's better to force the caller to handle the sentinel. Or maybe rename to `infer_candidate_subset`? From the outside, this will have both a `candidate_subset` property and a `get_candidate_subset` method, which is a little weird?

Reply:
good call -- it never gets used so I'll remove it